### ISSUE TITLE
SSL raises `NET::ERR_CERT_COMMON_NAME_INVALID` when any program tries to download master.tar.gz

### ISSUE THREAD

**Issue by hDLw08** [KSP-CKAN/CKAN#3705]
* Operating System: Windows 10 home 
* CKAN Version:  v1.31.0 or v1.31.2
* KSP Version: 1.12.4.3187 

**Have you made any manual changes to your GameData folder (i.e., not via **CKAN)?** no

Problem
----------
**Describe the bug**
When something updates there, I get this error (with CKAN v1.31.0, no error with v1.31.2)

**Steps to reproduce**
* Go to... **exe file CKAN**
* Click on...  **CKAN.exe**
* ~~See error~~ Stops responding during registry update

A web browser also fails to download the registry tar.gz file, with a `NET::ERR_CERT_COMMON_NAME_INVALID` error.

**Screenshots (if applicable)**

v1.31.2:

![image](https://user-images.githubusercontent.com/1559108/201486960-36df2ef1-6cd6-408f-aab8-7c2876086b39.png)

**CKAN error code (if applicable):**

> Failed to download https://github.com/KSP-CKAN/CKAN-meta/archive/master.tar.gz: The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.

**Comment by HebaruSan** [comment]
OK, the first suggestion would be to update to the latest version of CKAN:

https://github.com/KSP-CKAN/CKAN/releases/latest

There are many network-related changes since v1.31.0, which may or may not fix your problem, but there's no point in investigating old code.

**Comment by hDLw08** [comment]
In the new version, it crashes in a line "downloading https://github.com/KSP-CKAN/CKAN-meta/archive/master.tar.gz"

**Comment by HebaruSan** [comment]
Crashes, really? Can you please show us a screenshot of what that looks like so we can figure out what's happening?

**Comment by hDLw08** [comment]
fine, take it https://imgur.com/a/RJBP0Sj

**Comment by HebaruSan** [comment]
Thanks, but I meant the "crash" part. Are you actually seeing something that looks like a crash?

**Comment by hDLw08** [comment]
it gets stuck on it and crashes

**Comment by HebaruSan** [comment]
Why are you saying it "crashes", though? What are you seeing that leads you to conclude that?

**Comment by hDLw08** [comment]
CKAN is not clickable, but if you click on it, it freezes permanently

**Comment by HebaruSan** [comment]
OK, so it just stops responding, as opposed to displaying an error message or disappearing or any of the many other possible meanings of "crash"? Thanks.

**Comment by HebaruSan** [comment]
So far it sounds like your computer may be missing one or more SSL certificates. Are you able to download the link by clicking in a browser?

https://github.com/KSP-CKAN/CKAN-meta/archive/master.tar.gz

**Comment by hDLw08** [comment]
I can, but only with VPN, without it my browser says it's "hackers"

**Comment by HebaruSan** [comment]
Can you please show us a screenshot of what you mean by "my browser says it's "hackers"" ? This would help to understand what might be wrong with your PC.

**Comment by hDLw08** [comment]
here https://imgur.com/a/Na0ESbU

**Comment by HebaruSan** [comment]
Thanks, that seems to confirm that something's wrong with your SSL setup somehow. The exact problem is not clear, but it's not impossible that you actually _have_ been hacked, and this is how you're discovering it:

https://kinsta.com/knowledgebase/net-err_cert_common_name_invalid/

Since this is a low level part of the operating system these days, your best bet is probably to wipe your system and reinstall your operating system. Sorry that sounds so drastic, but it's potentially pretty serious.

**Comment by HebaruSan** [comment]
Closing as this is not caused by CKAN and cannot be fixed by CKAN. You can continue posting replies here if you figure out more details about what's going on or what the fix is, in case future users have similar problems and might be helped by further info.

**Comment by hDLw08** [comment]
I solved this problem, can't say exactly what helped me, but here it is.

1: I cleared the browser history
 
2: I reset the SSL status here's how to do it

**Open the Start menu.
Find and open Internet Options.
In the dialog box that appears, select the Content tab.
Click Clear SSL Status.**

3: I enabled enhanced protection in Chrome

4: I checked the PC for viruses with an antivirus.

I hope it helped if you also have a problem
